### ISSUE TITLE
开发人员文档门户网站的引言章节由我翻译， 翻译时间为2018.10.20-2018.10.21

### ISSUE THREAD

**Issue by bettybao1209** [bitshares/dev.bitshares.works#61]
Introduction & Architectures，Development Environments章节由我翻译，翻译时间为2018.10.20-2018.10.21。
LINK：http://dev.bitshares.works/en/master/intro/architectures.html
           http://dev.bitshares.works/en/master/intro/environments.html

**Comment by bt666** [comment]
此章节我已翻译完成，见PR#62和PR#63

**Comment by cedar-book** [comment]
I do not understand. Please English.

**Comment by bt666** [comment]
claim the translation task of these two chapters: Introduction & Architectures and Development Environments.
Original Link: http://dev.bitshares.works/en/master/intro/architectures.html
http://dev.bitshares.works/en/master/intro/environments.html

************************************************************************************
I've finished the task, refer to PR #62 and PR #63

**Comment by cedar-book** [comment]
Did you mean you created Chinese version of the documentation pages?

**Comment by bt666** [comment]
> Did you mean you created Chinese version of the documentation pages?

yes,  I'm requested to create issues in this way to claim the  task
